config.py

```diff
@@ -59,8 +60,11 @@ def main():
         else:
             args.pckgidir = '$SCRATCH'
 
+    if not args.stgidir:
+        args.stgidir = '$SCRATCH'
+
     config_data['config']['install_tree'] = args.pckgidir + '/spack-install/' + args.machine.replace('admin-', '')
-    config_data['config']['build_stage'] = [args.pckgidir + '/spack-stages/' + args.machine]
+    config_data['config']['build_stage'] = [args.stgidir + '/spack-stages/' + args.machine]
```

Comment:
@elsagermann I think we can use the same args.stgidir for both admin and non admin because there is args.machine in the path so it will be different for admin and non admin they will not overwrite each other

Reply:
Yes that should be fine, did you just check that Cosmo_PR, Spack_PR and dycore_PR set this -s new args?